js/data.json

```diff
@@ -156,7 +156,7 @@ var lp = {
 
 		{"id":"launchpad","category":"development","name":"Launchpad","description":"platform","address":"http://launchpad.net/","licenses":["gnu-agpl-v3"],"source":"https://dev.launchpad.net/"},
 		{"id":"codingteam","category":"development","name":"CodingTeam","description":"platform","address":"http://codingteam.net/","licenses":["gnu-agpl-v3"],"source":"http://codingteam.net/project/codingteam"},
-		{"id":"gitorious","category":"development","name":"Gitorious","description":"repositories","address":"http://gitorious.org/","licenses":["gnu-agpl-v3"],"source":"http://gitorious.org/gitorious"},
+		{"id":"gitlab","category":"development","name":"GitLab","OSS to build software":"repositories","address":"https://about.gitlab.com/","licenses":["mit"],"source":"https://gitlab.com/gitlab-org/gitlab-ce","introduction":"Create, review and deploy code together"},

```

Comment:
Looks good except for the replacement of »description« with »OSS to build software« because it breaks the json. If you change that back it looks good. »Repositories« is fine as description because it needs to be a very short tagline – except you can come up with a better one which is as short.